packages/altair-app/src/app/modules/altair/config.ts

```diff
@@ -75,6 +81,17 @@ export interface AltairConfigOptions {
    * Initial app settings to use
    */
   initialSettings?: Partial<fromSettings.State>;
+
+  /**
+   * Initial subscriptions provider
+   *
+   * @default "websocket"
+   */
+  initialSubscriptionsProvider?:
```

Comment:
If they are all imported from a single file, typing for this should also be easy, instead of having to specify the union of the types here.